src/main/kotlin/com/hibob/academy/service/SessionService.kt

```diff
@@ -0,0 +1,29 @@
+package com.hibob.academy.service
+
+import com.hibob.academy.resource.SessionApi.User
+import io.jsonwebtoken.Jwts
+import io.jsonwebtoken.SignatureAlgorithm
+import org.springframework.stereotype.Component
+import java.time.LocalDateTime
+import java.time.ZoneOffset
+import java.util.*
+
+@Component
+class SessionService {
+    companion object{
+        const val SECRET_KEY = "ndjiaqsfghouhm24vvycgjhfuytuytgiyusasfdghjkhgfdfgyuiuytrfftyuiuytrtyuihttixicuvb785v68568ig5buitui8onkgy7498cqdehqiwfhbui5p3hqugo5hr"
+    }
+
+    //The SessionApi.kt did not recognize this func when she was out of this block. why?
```

Comment:
wdym?

Reply:
I wrote this for me but dont remember what it mean by now. I think that the createJwtToken was need to be inside the companion object block